Sources/SIL/SIL.swift

```diff
@@ -38,7 +38,7 @@ public class Function {
 }
 
 // https://github.com/apple/swift/blob/master/docs/SIL.rst#basic-blocks
-public class Block: Equatable {
+public final class Block: Equatable {
```

Comment:
This needs to be `final` or otherwise I won't be able to conform `Block` to `CanSubstituteValueNames`, because then Swift complains that there's no guarantee that `Self` is actually a `Block` (which it really is, but I guess it would like the type to always get refined to a potential subclass...).